Lib/test/test_asyncio/test_tasks.py

```diff
@@ -2624,6 +2626,7 @@ class CTask_CFuture_Tests(BaseTaskTests, SetMethodsTest,
 
     Task = getattr(tasks, '_CTask', None)
     Future = getattr(futures, '_CFuture', None)
+    all_tasks = getattr(tasks, '_c_all_tasks', None)
```

Comment:
do we need to add a `hasattr(tasks, '_c_all_tasks`)` to the `skipUnless` decorator above?

Reply:
Well if `_CTask` exists then `_c_all_tasks` would exist anyways